packages/react-core/src/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -120,6 +121,35 @@ import FlagIcon from '@patternfly/react-icons/dist/esm/icons/flag-icon';
 ```ts file='./DescriptionListDefaultInlineGrid.tsx'
 ```
 
+### Display size
```

Comment:
Referencing this comment once more as it got lost after many updates

https://github.com/patternfly/patternfly-react/pull/7713#discussion_r926841741